### ISSUE TITLE
Crash Report

### ISSUE THREAD

**Issue by rickninja765** [Tobi823/ffupdater#397]
Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.NetworkException: File was not downloaded: /storage/emulated/0/Android/data/de.marmaro.krt.ffupdater/files/Download/com_kiwibrowser_browser_4736345870.apk
	at de.marmaro.krt.ffupdater.installer.ApkChecker$Companion.throwIfDownloadedFileHasDifferentSize(ApkChecker.kt:29)
	at de.marmaro.krt.ffupdater.DownloadActivity.startDownload(DownloadActivity.kt:315)
	at de.marmaro.krt.ffupdater.DownloadActivity.access$startDownload(DownloadActivity.kt:61)
	at de.marmaro.krt.ffupdater.DownloadActivity$startDownload$1.invokeSuspend(DownloadActivity.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:240)
	at android.os.Looper.loop(Looper.java:351)
	at android.app.ActivityThread.main(ActivityThread.java:8380)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.14 (140) release |
| Device | RMX3370 (RMX3370, RE879AL1, kona) |
| Manufacturer | realme (realme) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | dg02-pool03-kvm8, root, release-keys, 1677509287000 |

**Comment by nick-77** [comment]
Getting this same error

**Comment by Tobi823** [comment]
Fixed in 78.0.1. If not, please create a new issue.
